cost/azure/schedule_instance/README.md

```diff
@@ -0,0 +1,74 @@
+
+## Schedule Tag Example
+
+Start and Stop hours are 24-hour format: for example 8:15-17:30 is start at 8:15am, and stop at 5:30pm.
+
+Days of the week: SU, MO, TU, WE, TH, FR, SA
+
+Timezone: Use the TZ database name from the timezone list. For example, use America/New_York for Eastern time.
+
+Example: schedule=8:15-17:30;MO,TU,WE,TH,FR;America/New_York. Starts instances at 8:15am, stops instance at 5:30pm, Monday - Friday, Eastern Time.
+
+Instances are off during the weekend and start back up on Monday morning at 8:15am and are off at 5:30pm every weekday. Times are UTC unless the Timezone field is provided.
+
+Note: On leaving the minute field blank, policy will consider the minute as `00`
+and same will be added to the schedule label value.
+
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Email addresses* - A list of email addresses to notify  
+- *Exclusion Tags* - List of tags that an Instance can have to exclude it from the list. Format: Key=Value.
+- *Automatic Actions(s)* - (Optional)When this value is set, this policy will automatically take the selected action(s).
+
+## Policy Actions 
+
+The following policy actions are taken on any resources found to be out of compliance.
+ 
+- Send an email report  
+- stop - stop a selected instance
+- start - start a selected instance
+- terminate - terminates or deletes the selected instance.
+- update schedule - change existing schedule tag.  input to provide a new stop/start schedule
+- delete schedule - removes the schedule tag
+
+## Prerequisites
+
+### Schedule Label Format
+
+This policy uses `schedule` tag value for scheduling the instance. The format should be like `8:15-17:30;MO,TU,WE,TH,FR;America/New_York`. Please refer to `Schedule Tag Example` section for more details.
+On leaving the minute field blank, policy will consider the minute as `00` and same will be added to the schedule tag value.
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy, you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin, and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.   
+
+### Credential configuration
+
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
+
+Provider tag value to match this policy: `azure`
+
+Required permissions in the provider:
+
+- Microsoft.Compute/virtualMachines/read
```

Comment:
Is there a permission to modify the tags?

Reply:
Microsoft.Compute/virtualMachines/write --Creates a new virtual machine or updates an existing virtual machine

To add and update tags we are calling update VM API.